recipes/cspice/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+import time
+
+from conans import ConanFile, CMake, tools
+
+cspice_license = """THIS SOFTWARE AND ANY RELATED MATERIALS WERE CREATED BY THE
```

Comment:
where did you find this license? can you download it by `tools.download` instead?

Reply:
I've copied the last part at this url: https://naif.jpl.nasa.gov/naif/rules.html
vcpkg did the same thing.

Reply:
I see, could you please, split it from the recipe? You can create the file CSPICE_LICENSE.txt.


    exports = "CSPICE_LICENSE.txt"


Conan hooks doesn't allow files with the name "LICENSE", so you need to use that prefix.

Reply:
Ok. And what should be written for the license field in conanfile.py, something like "see CSPICE_LICENSE.txt"?

Reply:
Conan hooks also prevents exporting CSPICE_LICENSE.txt:
```
[HOOK - conan-center.py] pre_export(): ERROR: [EXPORT LICENSE (KB-H023)] This recipe is exporting a license file. Remove CSPICE_LICENSE.txt from `exports_sources` (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H023)
```

Other option: In the same base url than the downloaded zip file, there is a README file (https://naif.jpl.nasa.gov/pub/naif/misc/toolkit_N0066/C/PC_Cygwin_GCC_32bit/packages/README) where the license can be extracted.

Reply:
> And what should be written for the license field in conanfile.py

    license = "TSPA"

"SPICE software has been designated "Technology and Software Publicly Available" (TSPA) for export control purposes."

> Conan hooks also prevent exporting CSPICE_LICENSE.txt:

Rename it to TSPA.txt. The problem is the word "license".